modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -40,7 +40,10 @@
       purge   => true,
       force   => true,
       backup  => false,
-      source  => 'puppet:///modules/ocf_desktop/xsession/xsessions'
+      source  => 'puppet:///modules/ocf_desktop/xsession/xsessions';
+    # application default settings
+    '/etc/X11/Xresources/x11-ocf':
+      source => 'puppet:///modules/ocf_desktop/xsession/x11-ocf';
```

Comment:
if there's only one line maybe just use `content => "XScreenSaver*newLoginCommand:	xfce4-session-logout\n"` rather than adding an entire file. your call.